Makefile

```diff
@@ -1,5 +1,9 @@
+PREFIX          = /home/david/src/xLights/local
```

Comment:
I don't think it is good to have your user folder hard coded in the makefile

Reply:
Thanks for pointing that out, I missed resetting it back to the default.